doc/whats-new.rst

```diff
@@ -24,6 +24,20 @@ New Features
 Breaking changes
 ~~~~~~~~~~~~~~~~
 
+- The default backend ``engine`` used by :py:meth:`Dataset.to_netcdf`
+  and :py:meth:`DataTree.to_netcdf` is now chosen consistently with
+  :py:func:`open_dataset` and :py:func:`open_datatree`, using whichever netCDF
+  libraries are available and valid, and preferring netCDF4 to h5netcdf to scipy
+  (:issue:`10654`). This will change the default backend in some edge cases
+  (e.g., from scipy to netCDF4 when writing to a file-like object or bytes). To
+  avoid around these new defaults, set ``engine`` explicitly.
```

Comment:
Stumbled over this phrase, should it read, "to avoid these new defaults" or "to work around these new defaults"? Maybe "to bypass" or "to override"?